src/ContextMenuTrigger.js

```diff
@@ -113,11 +116,12 @@ export default class ContextMenuTrigger extends Component {
 
         hideMenu();
 
+        let id = typeof this.props.id === 'function' ? this.props.id(event) : this.props.id;
```

Comment:
`event` in React is a [`SyntheticEvent` and uses pooling](https://reactjs.org/docs/events.html#event-pooling).

This might not be an ideal approach

Reply:
mh I know for my case I do not actually need it since I can access  the selection of the `handontable` instances via the reference object. However would it be possible to give just the `target` object of the event like `this.props.id(event.target)`?